.github/project-labels.yml

```diff
@@ -0,0 +1,142 @@
+- name: component:infrastructure
+  color: "d93f0b"
+  description: >
+    The Issue/PR deals with development aiding infrastructure like CI,
+    distribution or helper tools.
+- name: component:internal
+  color: "d93f0b"
+  description: >
+    The Issue/PR deals with any project component that has no explicit
+    `component` label.
+- name: component:network
+  color: "d93f0b"
+  desscription: >
+    The Issue/PR deals with TCP/IP networking and establishing a test network
+    infrastructure.
+- name: component:technical docs
+  color: "d93f0b"
+  description: >
+    The Issue/PR deals with technical documentation like API, style, workflow
+    or similar documentation.
+- name: component:translation
+  color: "d93f0b"
+  description: >
+    The Issue/PR deals with the game terminology and the translation process
+    in general.
+- name: component:UI
+  color: "d93f0b"
+  description: >
+    The Issue/PR deals with the game interface, a consistent representation
+    style in-game.
+- name: component:user manual
+  color: "d93f0b"
+  description: >
+    The Issue/PR deals with end-user documentation targeted towards players and
+    server hosters.
+- name: priority:high
+  color: "2ab255"
+  description: >
+    The Issue/PR is very urgent or important and should be addressed/finished
+    as soon as possible.
+- name: priority:low
+  color: "2ab255"
+  description: >
+    The Issue/PR is of minor importance/urgency and can be set aside for more
+    urgent tasks.
+- name: status:abandoned
+  color: "c5def5"
+  description: The contributor providing the PR has stopped working on it.
+- name: status:cherry-pick for release
+  color: "c5def5"
+  description: The PR should be applied to the currently open release branch.
+- name: status:duplicate
+  color: "c5def5"
+  description: >
+    The topic of the Issue/PR is or was already covered by another Issue/PR.
+- name: status:superseded
+  color: "c5def5"
+  description: >
+    The PR implementation is insufficient and a follow up implementation PR
+    exists.
+- name: status:help wanted
+  color: "c5def5"
+  description: >
+    Other developers or external contributors are welcome to take over this
+    Issue/PR.
+- name: status:invalid
+  color: "c5def5"
+  description: >
+    The Issue is not classified as a formal valid Issue report or applicable PR.
+- name: status:merged
+  color: "c5def5"
+  description: >
+    All relevant commits of this PR were merged into the master development
+    branch.
+- name: status:resolved
+  color: "c5def5"
+  description: >
+    The Issue was resolved, either by answering properly or fixing the
+    underlying bug.
+- name: status:testing requested
+  color: "c5def5"
+  description: >
+    The Implementation can't be tested sufficiently by the developer and
+    require support.
+- name: status:won't fix
+  color: "c5def5"
+  description: >
+    The Issue won't be fixed as implementation or maintainance effort required
```

Comment:
can probably omit the (likely incomplete) list of reasons why

Reply:
I copied the description from the previous documentation and I tried to condense it to the essentials, which is usually "it's too much effort for the gain".